src/dask_histogram/boost.py

```diff
@@ -85,8 +85,17 @@ def __init__(
         """Construct a Histogram object."""
         super().__init__(*axes, storage=storage, metadata=metadata)
         self._staged: AggHistogram | None = None
-        self._dask_name: str | None = None
-        self._dask: HighLevelGraph | None = None
+        self._dask_name: str | None = (
+            f"empty-histogram-{tokenize(*axes, storage, metadata)}"
+        )
+        self._dask: HighLevelGraph | None = HighLevelGraph(
+            {
+                self._dask_name: {
+                    (self._dask_name, 0): (lambda: self._in_memory_type(self),)
```

Comment:
I don't know this code well, so just double checking: does `self._in_memory_type(self)` make sense, and is the comma following it intentional?

Reply:
it wanted a tulple.

Ah, I might be able to get rid of the `self._in_memory_type` indeed.

Reply:
And yes, in case there are no partitions in the histogram make sure to return an empty histogram on compute.

The "empty-histogram" taskgraph is overwritten as soon as any `fill` call is made.